server/controllers/DataAccessLayer.js

```diff
@@ -1,9 +1,27 @@
 const fs = require('fs');
 const lineReader = require('n-readlines');
 const replace = require('replace-in-file');
+
 let User = require("../model/User.js");
+let Report = require("../model/Report.js");
+let ChatMessage = require("../model/ChatMessage.js");
 
 let cachedUsers = [];
+let cachedReports = [];
+
+/**
+ * Retrieves the array of cached Reports
+ */
+exports.getCachedReports = function() {
+    return cachedReports;
+}
+
+/**
+ * Retrieves cached users
+ */
+exports.GetCachedUsers = function() {
```

Comment:
I think I've added a similar function that is pushed to master

Reply:
This is the master one, I moved it.

Reply:
Nevermind, I saw that you just moved it to the top